styles/layout.less

```diff
@@ -1497,6 +1497,11 @@ table.search-column {
     }
 }
 
+#user-filterbox{
+    #linkbox;
+    padding: 0.5em 0.5em;
```

Comment:
Do we really need the padding tweak? This makes it a snowflake with all the other uses of this type of floating box.

Reply:
The other linkboxes work around the spacing by having a `<p>` inside the `<form>` which introduces extra padding. It actually ends up being more padding than the `0.5em` I added here.

In early iterations of this PR that looked like a bit much, but since the box is now at the top right of the page, it doesn't really matter, so I've removed the special case `user-filterbox`.